lineman/app/components/start_menu/start_menu.haml

```diff
@@ -0,0 +1,9 @@
+.lmo-start-menu
```

Comment:
Right now the convention we're following is we only prefixing lmo onto utility classes. Can we follow that here? or there something clashing with start-menu?